.github/workflows/update-eventuals.yml

```diff
@@ -0,0 +1,17 @@
+name: Create a PR for updating eventuals in `3rdparty/eventuals-tutorial` repo
+
+on:
+  push:
+    branches:
+      - "artur.pr-bot-for-updating-eventuals"
+  workflow_dispatch:
+
+jobs:
+  PR-bot-for-updating-eventuals:
+    uses: 3rdparty/dev-tools/.github/workflows/update-eventuals-bot.yml@artur.PR-bot-for-updating-eventuals
```

Comment:
Let's just make this job have all of this because this is eventuals specific so let's not store in dev-tools.